src/VisualStudio/VisualStudioInteractiveComponents/source.extension.vsixmanifest

```diff
@@ -4,13 +4,35 @@
     <Identity Id="500fff63-afcf-4195-8db4-3fa8a5180e79" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Interactive Components</DisplayName>
     <Description>Interactive Roslyn components for Visual Studio.</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\VBCSharp\InteractiveComponents</ShortcutPath>
   </Metadata>
   <Installation Experimental="true">
     <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[14.0,]" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinDesktopExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
   </Installation>
+  <Installer>
+    <Actions>
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.InteractiveEditorFeatures.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.Scripting.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Features.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.InteractiveEditorFeatures.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.InteractiveFeatures.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Scripting.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Workspaces.Desktop.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Workspaces.dll" />
+      <Action Type="Ngen" Path="Microsoft.VisualStudio.InteractiveServices.dll" />
+      <Action Type="Ngen" Path="Roslyn.VisualStudio.InteractiveComponents.dll" />
+      <Action Type="Ngen" Path="System.Collections.Immutable.dll" />
+      <Action Type="Ngen" Path="System.Diagnostics.StackTrace.dll" />
+      <Action Type="Ngen" Path="System.IO.FileSystem.dll" />
+      <Action Type="Ngen" Path="System.IO.FileSystem.Primitives.dll" />
+      <Action Type="Ngen" Path="System.Reflection.Metadata.dll" />
```

Comment:
I am not sure the System.* binaries like System.IO.* above need to be ngen-ed (since we would have been excluding them from the list at insertion-time before). It may be a good idea to check the exclusion list and only ngen binaries that we were actually dropping into PrivateAssemblies...

However, if this is harmless, I am also fine with leaving this in for now and coming back to clean this up later.